### DISCUSSION TITLE
Issues encountered after a fresh EOS 8 installaton

### DISCUSSION THREAD

**Discussion by vinzuk80** [elementary/triage#530] [Q&A] [answered]
Good morning, 

I'm totally new on Github, and don't know if it's the right place to post some minor issues I encountered after a fresh installation of Elementary OS 8. 
I enabled the wayland session, and out of the box, it seems pretty fine. 

- I have noticed sometimes the "app store" to freeze and crash while looking for flatpak apps. 
- I previously installed EOS 7.1, and using now EOS 8, I don't see the blue dot bellow an opened app. I did not find any parameter to enable it in the settings. 
- When an app is displayed, under EOS 7.1, when you click the app icone in the dock, it minimises it, but apparently not under EOS 8. 
- I tried to install some appimage, unsuccessfully. It's all right with EOS 7.1. I did the same steps by letting the appimage the right executable rights. 

Appart from those minor bugs, I fine EOS 8 very well polished and working really well. 
Thanks for your hard work, really appreciate it.

**Comment by danirabbit** [comment, marked as answer]
Hey thanks for your feedback! The best way to make sure the team can address your feedback efficiently is to open separate issue reports for each issue you encounter. There is an issue reporting guide here with helpful advice on how to file useful reports: https://docs.elementary.io/contributor-guide/feedback/reporting-issues

Gonna close this discussion post as resolved since there's a clear action path here :)
